salt/post_provisioning_cleanup.sh

```diff
@@ -14,6 +14,17 @@ fi
 
 # Nothing to do in case "install_salt_bundle" grain is not true
 INSTALL_SALT_BUNDLE=$(${SALT_CALL} --local --log-level=quiet --output=txt grains.get install_salt_bundle)
+SERVER=false
+ROLES=$(${SALT_CALL} --local --log-level=quiet --output=txt grains.get roles)
+if [[ "${ROLES}" == "local: ['server']" ]];then
```

Comment:
The `roles` variable is a actually a list, so in principle it could accept multiple values and therefore this comparision would fail.

Maybe something like:

```suggestion
if [[ "${ROLES}" == *"'server'"* ]];then
```
What do you think?